frontend/src/app/testQueue/TestCardForm/TestCardForm.utils.tsx

```diff
@@ -204,6 +211,15 @@ export const useAOEFormOption = (
   if (!hasAnySupportedDiseaseTests(testFormState.deviceId, devicesMap)) {
     return AOEFormOption.COVID;
   }
+  if (
+    devicesMap
+      .get(testFormState.deviceId)
+      ?.supportedDiseaseTestPerformed.filter(
+        (x) => x.supportedDisease.name === MULTIPLEX_DISEASES.GONORRHEA
+      ).length === 1
+  ) {
+    return resultHasPositive ? AOEFormOption.GONORRHEA : AOEFormOption.NONE;
```

Comment:
Hmm good point, we should make sure everyone on the team knows we can't support multi-STI devices yet. I don't think we necessarily need to add separate validation since the multiple device test card redesign is one of the big upcoming features that will likely require us to deal with those AOE questions in a more comprehensive manner

Reply:
agree! seems like this won't be the case for long